tutorials/buoyantSimpleFoam-aster/Allclean.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+rm -f precice-Fluid-convergence.log
+rm -f precice-Fluid-events.json
+rm -f precice-Fluid-iterations.log
+rm -f solid/solid.mess
+rm -f solid/solid.resu
+
+# Add file deletion for solid.rmed
+
+# pyFoamClearCase.py
```

Comment:
We don't need this line:

```suggestion
```

We can instead use the intrinsic OpenFOAM `cleanCase` script, if we first load it using:

```bash
# Source tutorial clean functions
. $WM_PROJECT_DIR/bin/tools/CleanFunctions
```

See an [example](https://github.com/precice/openfoam-adapter/blob/master/tutorials/CHT/flow-over-plate/buoyantPimpleFoam-laplacianFoam/Allclean).

Reply:
There is actually already an `Allclean` file inside the `Fluid/` case. In the rest of the tutorials we have the `Allclean` script in the same path as the `Allrun`.

Reply:
The file was renamed to `Allclean` and everything looks good there.